multigeiger/multigeiger.ino

```diff
@@ -506,7 +506,6 @@ void loop()
   update_display = (GMC_counts >= MAXCOUNTS) || ((current_ms - time2display) >= DISPLAYREFRESH);
   if(update_display) isr_GMC_counts = 0;
   count_timestamp = isr_count_timestamp;
-  count_timestamp_2send = isr_count_timestamp_2send;
```

Comment:
maybe have such stuff in a separate commit in future, it would be much cleaner that just sneaking such stuff into an otherwise unrelated commit.

also, one could in the end do a `git rebase -i master` and merge the commit that removed this line with the other commit that removed the other line, because these 2 changes are really strongly related.

Reply:
next time